mmocr/visualization/textspotting_visualizer.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Optional, Sequence, Union
+
+import mmcv
+import numpy as np
+import torch
+
+from mmocr.structures import TextDetDataSample
+from mmocr.utils.polygon_utils import poly2bbox
+from .base_visualizer import BaseLocalVisualizer
+
+
+class TextSpottingLocalVisualizer(BaseLocalVisualizer):
+
+    def _draw_instances(
+        self,
+        image: np.ndarray,
+        bboxes: Union[np.ndarray, torch.Tensor],
+        polygons: Sequence[np.ndarray],
+        texts: Sequence[str],
+    ) -> np.ndarray:
+        """Draw instances on image.
+
+        Args:
+            image (np.ndarray): The origin image to draw. The format
+                should be RGB.
+            bboxes (np.ndarray, torch.Tensor): The bboxes to draw. The shape of
+                bboxes should be (N, 4), where N is the number of texts.
+            polygons (Sequence[np.ndarray]): The polygons to draw. The length
+                of polygons should be the same as the number of bboxes.
+            edge_labels (np.ndarray, torch.Tensor): The edge labels to draw.
+                The shape of edge_labels should be (N, N), where N is the
+                number of texts.
+            texts (Sequence[str]): The texts to draw. The length of texts
+                should be the same as the number of bboxes.
+            class_names (dict): The class names for bbox labels.
+            is_openset (bool): Whether the dataset is openset. Default: False.
```

Comment:
del, also don't forget to sync the docstring after updating the counterpart in `KIEVisualizer`